qa/rpc-tests/test_framework/util.py

```diff
@@ -99,11 +108,31 @@ def sync_mempools(rpc_connections, wait=1):
 
 bitcoind_processes = {}
 
-def initialize_datadir(dirname, n):
+def initialize_datadir(dirname, n,bitcoinConfDict=None,wallet=None):
     datadir = os.path.join(dirname, "node"+str(n))
     if not os.path.isdir(datadir):
         os.makedirs(datadir)
+    
+    defaults = {"server":1, "discover":0, "regtest":1,"rpcuser":"rt","rpcpassword":"rt",
```

Comment:
With much structural changes going on, I'm not even sure if "blockprioritysize":2000000 is a newly added default or not.
It seemed to be, based on the addition in the commented out (`if 0:`) section elsewhere.

If it is truly an addition, please comment why this parameter is needed and why this value was chosen.